.gitignore

```diff
@@ -8,4 +8,11 @@ scripts/auto_gpt_workspace/*
 *.mpeg
 .env
 last_run_ai_settings.yaml
-outputs/*
\ No newline at end of file
+outputs/*
+/.idea/.gitignore
+/.idea/Auto-GPT-original.iml
```

Comment:
You have a leading slash that makes this not work -- and you can use glob patterns. For example:

```
.idea/*
```

or just

```
idea/
```